src/app/components/core/configuration-manager/view-config-item/view-config-item.component.ts

```diff
@@ -506,4 +509,14 @@ export class ViewConfigItemComponent implements OnInit, OnChanges, OnDestroy {
       }
     });
   }
+
+  navToLink() {
+    const v = version.includes('next') ? 'develop' : `v${version}`;
+    const p = `${name === 'fledge' ? 'fledge' : 'foglamp'}-${this.pluginInfo.type.toLowerCase()}-${this.pluginInfo.name.toLowerCase()}`;
+    if (name === 'fledge') {
```

Comment:
This will not work as white labeling job will replace fledge with foglamp

Reply:
name is coming from package.json so when it is replaced it will be picked here changed.

Reply:
Yes